WordPress/src/main/java/org/wordpress/android/widgets/WPPrefView.java

```diff
@@ -431,18 +419,14 @@ private Dialog getCheckListDialog() {
             checkedItems[i] = mListItems.get(i).mIsChecked;
         }
 
-        AlertDialog.Builder builder = new AlertDialog.Builder(
-                new ContextThemeWrapper(getContext(), R.style.Calypso_Dialog_Alert))
+        AlertDialog.Builder builder = new MaterialAlertDialogBuilder(getContext())
                 .setTitle(mTitleTextView.getText())
                 .setNegativeButton(android.R.string.cancel, null)
-                .setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        SparseBooleanArray checkedItems =
-                                ((AlertDialog) dialog).getListView().getCheckedItemPositions();
-                        mListItems.setCheckedItems(checkedItems);
-                        doPrefChanged();
-                    }
+                .setPositiveButton(android.R.string.ok, (dialog, which) -> {
+                    SparseBooleanArray checkedItems1 =
```

Comment:
Nitpick: Should we name `checkedItems1` to something else? Or maybe we could do something like this:

```
mListItems.setCheckedItems(((AlertDialog) dialog).getListView().getCheckedItemPositions());
```

_P.S: Android Studio can't find `MaterialAlertDialogBuilder` for me, so I couldn't play with it._

Reply:
Good catch! AS does it sometimes with optimizing code.